aspnetcore/getting-started.md

```diff
@@ -23,6 +23,8 @@ uid: getting-started
     cd aspnetcoreapp
     dotnet new web
     ```
+    
+    Note: If you get an error running `dotnet new web` or you must run  `dotnet new -t web`, you'll need to upgrade to the current version of .NET Core. Install [.NET Core](https://microsoft.com/net/core)
```

Comment:
"or you must run" isn't clear.  How about 

If you get an error running `dotnet new web`, don't change the command to `dotnet new -t web` to get it to work because that form of the command only works with an older version of .NET Core.  Install the current version of [.NET Core](https://microsoft.com/net/core).

Could shorten by omitting "because that form of the command only works with an older version of .NET Core" but that makes the issue a little clearer.